sql/core/src/test/resources/sql-tests/inputs/show-tblproperties.sql

```diff
@@ -1,26 +0,0 @@
--- create a table with properties
-CREATE TABLE tbl (a INT, b STRING, c INT) USING parquet
-TBLPROPERTIES('p1'='v1', 'p2'='v2');
-
-SHOW TBLPROPERTIES tbl;
-SHOW TBLPROPERTIES tbl("p1");
-SHOW TBLPROPERTIES tbl("p3");
-
-DROP TABLE tbl;
-
--- create a view with properties
-CREATE VIEW view TBLPROPERTIES('p1'='v1', 'p2'='v2') AS SELECT 1 AS c1;
```

Comment:
does the `ShowTblPropertiesSuiteBase` cover view?

Reply:
I think [here](https://github.com/apache/spark/blob/06586f681bb72c4ad48ef9203e290d75b2732670/sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/ShowTblPropertiesSuite.scala#L50) cover the view test. but just for v1.